test/functional/feature_nulldummy.py

```diff
@@ -20,13 +20,14 @@
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import assert_equal, assert_raises_rpc_error
 
+HEIGHT = 99
```

Comment:
it would be good to self-documenting name, as it is not clear which block height this refers to

Reply:
done